VAMobile/documentation/docs/Flagship design library/Templates/Home.md

```diff
@@ -0,0 +1,106 @@
+* **What lives in this zone:**
+    * [Encouraged update](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/App%20Features/EncouragedUpdate/) (MVP)
+    * [What’s new](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/App%20Features/WhatsNew/) (MVP)
+* **Guidelines for future module inclusion:**
+Information is appropriate for inclusion on the home screen within the mission-critical app info section when it:
+    * Conveys information that’s necessary for the app to function properly
+    * Conveys information about important changes within the app
+
+
+#### Zone 2: Activity 
+
+* **What it is:** An alphabetical list of feature modules giving Veterans an overview of what VA is doing for them—includes any critical “in-flight” tasks or actions that may need attention. Each module contains personalized information briefly summarizing that activity and provides a secondary entry point into the app feature.
+    * **Module appearance logic:** Variable. Modules appear while that feature’s activity is still considered “in flight” or when there is a user action to take. Each feature has different periods of time/circumstances that determine how long they are visible on the home screen (ex: the Claims module stays until the claim closes, the Secure Message stays until any new messages have been viewed).
+    * **Module anatomy**
+[image of module]
+        * **Title:** Feature name
+        * **Description text:** Indicates # of items with an in-flight status; terms used vary by feature; provides just enough information to accurately indicate what is happening/is needed. 
+* **What lives in this zone:**
+    * Prescriptions (MVP)
+    * Appointments (MVP)
+    * Messages (MVP)
+    * Claims (MVP)
+    * Debts *(future)*
+    * Letters & documents *(future)*
+    * Payments *(future)*
+    * Vaccine/medical records *(future)*
+    * Travel pay *(future)*
+* **Guidelines for module inclusion:**
+A feature is appropriate for inclusion as an Activity module on the home screen when it:
+    * Has an “in-flight” status (ex: an open Claim, an upcoming Appointment)
+    * Has an action that is available for the user to take (ex: a refillable Prescription, an unread Message)
+
+
+#### Zone 3: About you:
+
+* **What it is:** A personalized, high-level snapshot of the individual Veteran at VA. 
+    * **Module appearance logic:** Fixed appearance.
+* **What lives in this zone:**
+    * Branch of Service/Veteran Status card (MVP)
+    * Disability rating & monthly payment (MVP)
+* **Guidelines for module inclusion:**
+Information is appropriate for inclusion on the home screen within About you when it:
+    * <span style="text-decoration:underline;">Is</span> or <span style="text-decoration:underline;">provides access to</span> information relating to who the individual Veterans *is within VA* as opposed to relating to a specific activity or task
```

Comment:
Can we change these to some other type of emphasis? underlining non-links causes some a11y concerns

Reply:
yep